e2e/cookie-tests.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What is this?

Reply:
A file automatically generated by tests. Should it be added to gitignore?

Reply:
Probably, but which test package generates this?

Reply:
`e2e/scenarioe2e.spec.js`